components/private_channel/rust/ffi/tests/main.cc

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <array>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "private_channel.h"  // NOLINT
```

Comment:
please use the full path

Reply:
also tests need to be run as part of the gn test targets, not as custom test code